go-controller/cmd/ovn-kube-util/ovn-kube-util.go

```diff
@@ -41,7 +44,31 @@ func main() {
 		return nil
 	}
 
-	if err := c.Run(os.Args); err != nil {
+	ctx := context.Background()
+
+	// trap SIGHUP, SIGINT, SIGTERM, SIGQUIT and
+	// cancel the context
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
This is not "cancelling the context"... it's creating a cancellable context.

But this is all just cut+pasted anyway. It's in 3 places now. It should be moved to `pkg/util/` or something

Reply:
+1